trace-sdk/src/main/java/io/bitrise/trace/data/collector/crash/TraceExceptionDataListener.java

```diff
@@ -0,0 +1,55 @@
+package io.bitrise.trace.data.collector.crash;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.annotation.VisibleForTesting;
+import io.bitrise.trace.data.collector.BaseDataListener;
+import io.bitrise.trace.data.dto.CrashData;
+import io.bitrise.trace.data.management.formatter.crash.ExceptionDataFormatter;
+
+/**
+ * Listens for any unhandled exceptions that get thrown and passes them to the
+ * {@link ExceptionDataFormatter}.
+ */
+public class TraceExceptionDataListener extends BaseDataListener
```

Comment:
what do you think of `TraceExceptionDataListener` instead of `ExceptionDataListener`?